BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -669,4 +638,20 @@ public void onLongItemClicked(int position, BookItem bookItem, View v) {
     }
   }
 
+  private void shareShelfBibIntent() {
+
+    Uri contentUri = exportBibTex.writeTemporaryBibFile(context,
+        exportBibTex.getBibDataFromShelf(shelfId, bookDao, noteDao));
+
+    Intent shareShelfIntent =
+        ShareCompat.IntentBuilder.from(getActivity())
+            .setStream(contentUri)
+            .setType("text/*")
```

Comment:
Maybe use a variable for "text/*" because you also use it in LibraryFragment.

Reply:
See comment below.